cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1724,6 +1725,176 @@ def test_new_storage_multiple_subnets_validator(queues, new_storage_count, failu
+                        subnet_ids=["subnet-2", "subnet-3"],
+                    ),
+                ),
+            ],
+            ["subnet-1", "subnet-2"],
+            FailureLevel.INFO,
+            "Your configuration for Queue multi-az-queue-2 includes multiple subnets and external "
+            "shared storage configuration. Accessing a shared storage from different AZs can lead to increased "
+            "latency and costs.",
+        ),
+        (
+            [
+                SlurmQueue(
+                    name="multi-az-queue-1",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1", "subnet-2"],
+                    ),
+                ),
+                SlurmQueue(
+                    name="multi-az-queue-2",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-2", "subnet-3"],
+                    ),
+                ),
+            ],
+            ["subnet-2"],
+            FailureLevel.INFO,
+            "Your configuration for Queue multi-az-queue-1 includes multiple subnets and external "
+            "shared storage configuration. Accessing a shared storage from different AZs can lead to increased "
+            "latency and costs.",
+        ),
+        (
+            [
+                SlurmQueue(
+                    name="multi-az-queue-1",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1", "subnet-2"],
+                    ),
+                ),
+                SlurmQueue(
+                    name="multi-az-queue-2",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-2", "subnet-3"],
+                    ),
+                ),
+            ],
+            ["subnet-2"],
+            FailureLevel.INFO,
+            "Your configuration for Queue multi-az-queue-2 includes multiple subnets and external "
+            "shared storage configuration. Accessing a shared storage from different AZs can lead to increased "
+            "latency and costs.",
+        ),
+        (
+            [
+                SlurmQueue(
+                    name="multi-az-queue-1",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1", "subnet-2"],
+                    ),
+                ),
+                SlurmQueue(
+                    name="multi-az-queue-2",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1", "subnet-2"],
+                    ),
+                ),
+            ],
+            ["subnet-1", "subnet-2"],
+            None,
+            "",
+        ),
+    ],
+)
```

Comment:
Can you please make one of these cases use the `SchedulerPluginQueueNetworking` class?

Reply:
Fixed